### PR TITLE
Fixing up the casing of spec defined terms.

### PR DESCRIPTION
**Pull request by n3wscott**
As we talked about in the SIG.

### CODE REVIEW

**Comment by angarg12** [comment]
LGTM

**Comment by duglin** [comment]
LGTM

**Comment by duglin** [comment]
@angarg12 needs your re-review

**Comment by avade** [comment]
lgtm

**Comment by fmui** [comment]
LGTM

File: profile.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ the [Open Service Broker API specification](spec.md) SHALL take precedence.
   interact with Service Brokers to provision, and manage, Service Instances
   and Service Bindings.
 
-- *Service Instance*: An instantiation of a Service offering.
+- *Service Instance*: An instantiation of a Service Offering.
```
Comment (vaikas-google) [inline profile.md]: I'd expect 'Service Offering' then to be defined somewhere? Maybe capitalize on L45 as well then?

**Comment by angarg12** [comment]
Now this PR needs rebase. Maybe we can also address @vaikas-google comment at the same time.

File: profile.md (hunk 1 above)
Comment (n3wscott) [reply profile.md]: It is defined in the spec and the terms defined in profile are now pointed to the spec.

**Comment by n3wscott** [comment]
@angarg12 Rebased. Thanks!

**Comment by angarg12** [comment]
LGTM

**Comment by duglin** [comment]
LGTM

2 more reviews needed

**Comment by mattmcneeney** [comment]
This is going to cause me a lot of pain with #159 and #334... but I support this so... _gulp_...
LGTM

**Comment by n3wscott** [comment]
@mattmcneeney :) sorry !! I can help you if you need with those PRs

**Comment by fmui** [comment]
LGTM

**Pull request merged**